.github/workflows/update_examples_pr_workflow.yml

```
@@ -0,0 +1,47 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+# This is a basic workflow to help you get started with Actions
+
+name: "Powershell Update Examples Files PR"
+# Controls when the action will run. Triggers the workflow on push or pull request
+# events but only for the dev branch
+on:
+  push:
+    branches: 
+      - 'Weekly_examples_update_*'
```

Let's stick with existing branch naming convention of `{ChangeName}/{TimeStamp}`. See https://github.com/microsoftgraph/msgraph-sdk-powershell/blob/dev/.github/workflows/create-openapidocs-pr-workflow.yml#L11.
```suggestion
      - 'WeeklyExamplesUpdate/*'
```